Code/Framework/AzFramework/AzFramework/SurfaceData/SurfaceData.h

```diff
@@ -17,6 +17,7 @@ namespace AzFramework::SurfaceData
     namespace Constants
     {
         static constexpr const char* s_unassignedTagName = "(unassigned)";
+        static constexpr AZ::Crc32 s_unassignedTagCrc = AZ::Crc32(Constants::s_unassignedTagName);
```

Comment:
Coding standards say these should be named `UnassignedTagCrc` and `UnassignedTagName` 
https://github.com/o3de/sig-core/blob/main/governance/Coding-Standards-and-Style-Guide.md#constant-names